### DISCUSSION TITLE
Older Cam Pan v2 firmware?

### DISCUSSION THREAD

**Discussion by rkrueger11** [gtxaspec/wz_mini_hacks#444] [Q&A]
I'm on the latest firmware, but want RTSP capabilities.. anyone have the older 4.36.9.139 firmware or before saved? It's no longer available on Wyze.com.

**Comment by daniel-lacour** [comment]
I'd like to know if 4.36.9.139 applies to the V2 pan too... It looks like It doesn't, but I'm not sure.

**Reply by Bob-AOL** [reply]
4.36.9.139 is what my newest v3s upgraded to and is basically now a roll back for their newest release which had issues. It is not RTSP firmware and I don't believe it applies to your request. 4.61.0.3 is the RTSP firmware that would be compatible with a 4.36.9.139 camera.

You may be quite limited in your choices for PanCam v2 RTSP capable firmware.

**Comment by Bob-AOL** [comment]
#351 may apply.
